client/my-sites/subscribers/components/add-subscribers-modal/add-subscribers-modal.tsx

```diff
@@ -18,6 +18,13 @@ const AddSubscribersModal = ( { siteId, siteTitle }: AddSubscribersModalProps )
 	const { showAddSubscribersModal, setShowAddSubscribersModal, addSubscribersCallback } =
 		useSubscribersPage();
 
+	useEffect( () => {
+		// Open "add subscribers" modal by default via URL
```

Comment:
I'm not super happy about this placement, and was hoping to put in `main.jsx` instead which is the top level rendering component for the route.

Thoughts?